### ISSUE TITLE
'Restore Default Configuration' gives error with single screen

### ISSUE THREAD

**Issue by jcpearson** [tb2097/wacom-gui#6]
Selecting 'Other Setting -> Restore Default Configuration' gives the error:

```
Traceback (most recent call last):
  File "wacom-gui.py", line 109, in restoreDefaults
    self.options.resetDefaults()
  File "/tmp/wacom-gui-master/wacom-gui/options.py", line 203, in resetDefaults
    self.screenFull.setChecked(True)
AttributeError: 'otherOptions' object has no attribute 'screenFull'
```
I guess you need to check multiple screens are in use or self.screenFull exists before attempting self.screenFull.setChecked(True) ?

**Comment by tb2097** [comment]
Fixed in V0.2.0e2

**Comment by jcpearson** [comment]
Thanks - works fine now
